app/lib/features/chat/pages/room_page.dart

```diff
@@ -253,7 +254,7 @@ class _ChatRoomConsumerState extends ConsumerState<ChatRoom> {
           inputPlaceholder: L10n.of(context).message,
           sendButtonAccessibilityLabel: '',
         ),
-        timeFormat: DateFormat.jm(),
+        timeFormat: is24HoursFormat ? DateFormat.Hm() : DateFormat.jm(),
```

Comment:
so, I see... it looks like DateFormat isn't set up to use the right `local` ... shouldn't we rather be fixing that problem then?